app/views/bets/_bet.html.erb

```diff
@@ -0,0 +1,7 @@
+<tr>
+  <td class="col-md-2"><%= bet.creator_id %></td>
+  <td class="col-md-2"><%= bet.created_at %></td>
+  <td class="col-md-2"><%= bet.expires_at %></td>
+  <td class="col-md-2"><%= bet.bet_total %></td>
+  <td class="col-md-4"><%= bet.bet %></td>
```

Comment:
Probably worth doing at this point. You're the third person to point this out.

Would be a good time to learn the `sed` bash command